packages/api/src/commands/install.ts

```diff
@@ -23,19 +90,8 @@ cmd
     new Option('-l, --lang <language>', 'SDK language').default(SupportedLanguages.JS).choices([SupportedLanguages.JS]),
   )
   .addOption(new Option('-y, --yes', 'Automatically answer "yes" to any prompts printed'))
-  .action(async (uri: string, options: { identifier?: string; lang: SupportedLanguage; yes?: boolean }) => {
-    let language: SupportedLanguage;
-    if (options.lang) {
-      language = options.lang;
-    } else {
-      ({ value: language } = await promptTerminal({
-        type: 'select',
-        name: 'value',
-        message: 'What language would you like to generate an SDK for?',
-        choices: [{ title: 'JavaScript', value: SupportedLanguages.JS }],
-        initial: 1,
-      }));
-    }
+  .action(async (uri: string, options: Options) => {
+    const language = await getLanguage(options);
```

Comment:
Just some very small refactors to move this `language` and `identifier` selection out so this command action is a little easier to read.